src/frontends/tensorflow/tests/convert_unsupported.cpp

```diff
@@ -32,9 +33,34 @@ TEST(FrontEndConvertModelTest, test_unsupported_op) {
     ASSERT_THROW(frontEnd->convert(function), OpConversionFailure);
 
     for (auto& node : function->get_ordered_ops()) {
-        if (node->get_friendly_name() == "relu_0" && std::dynamic_pointer_cast<ov::op::util::FrameworkNode>(node)) {
-            function->replace_node(node, std::make_shared<opset6::Relu>(node->input(0).get_source_output()));
+        if (node->get_friendly_name() == "relu_0" && dynamic_pointer_cast<ov::op::util::FrameworkNode>(node)) {
+            function->replace_node(node, make_shared<opset6::Relu>(node->input(0).get_source_output()));
         }
     }
     ASSERT_NO_THROW(frontEnd->convert(function));
 }
+
+TEST(FrontEndConvertModelTest, test_unsupported_tf1_while) {
+    FrontEndManager fem;
+    FrontEnd::Ptr frontEnd;
+    InputModel::Ptr inputModel;
+    ASSERT_NO_THROW(frontEnd = fem.load_by_framework(TF_FE));
+    ASSERT_NE(frontEnd, nullptr);
+    auto model_filename = FrontEndTestUtils::make_model_path(string(TEST_TENSORFLOW_MODELS_DIRNAME) +
+                                                             string("model_tf1_while/model_tf1_while.pb"));
+    ASSERT_NO_THROW(inputModel = frontEnd->load(model_filename));
+    ASSERT_NE(inputModel, nullptr);
+    shared_ptr<ngraph::Function> function;
+
+    try {
+        function = frontEnd->convert(inputModel);
+        FAIL() << "TensorFlow 1 While must not be supported now.";
```

Comment:
<< "TensorFlow 1 While is not supported in TF FE but conversion passed without errors. OpConversionFailure is expected."

Reply:
Fixed.